mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -1719,10 +1719,10 @@ msgid "dodge_increased"
 msgstr "{stat} de {user} augmenté."
 
 msgid "attack_resisted"
-msgstr "L'attaque a résisté !"
+msgstr "L'adversaire a résisté à l'attaque !"
 
 msgid "status_overfeed"
-msgstr "Sur-alimenter"
+msgstr "Trop mangé"
```

Comment:
I'm not sure about what is this status about, but I don't really like this expression... It's like not an expression.
Wouldn't "Suralimentation / Suralimenté " be fine? Looks like it's a valid word https://www.larousse.fr/dictionnaires/francais/suralimentation/75594

If it's about a nature, it could be "Glouton", or "Goinfre", but yeah in this case I guess it's really only about a temporary status...

Reply:
What about "Repu"?

Reply:
How didn't I think about this one!